builtin/credential/cert/path_certs.go

```diff
@@ -114,6 +114,14 @@ formatted as "oid:value". Expects the extension value to be some type of ASN1 en
 All values much match. Supports globbing on "value".`,
 			},
 
+			"allowed_metadata_extensions": {
+				Type: framework.TypeCommaStringSlice,
+				Description: `A comma-separated string or array of oid extensions.
```

Comment:
Does a comma separated string work here ? I didn't see a split by comma way of parsing the input in the code, hence figured i'd check.

Reply:
ugh i missed the `TypeCommaStringSlice` there, please ignore!